src/components/autocomplete/js/autocompleteDirective.js

```diff
@@ -397,7 +393,45 @@ function MdAutocomplete ($$mdSvgRegistry) {
             ? '<li ng-if="$mdAutocompleteCtrl.notFoundVisible()"\
                          md-autocomplete-parent-scope>' + template + '</li>'
             : '';
+      }
+
+      function getContainerType(attr) {
+        var menuContainerClass = attr.mdMenuContainerClass ? ' ' + attr.mdMenuContainerClass : '';
+
+        if (isVirtualRepeatDisabled(attr)) {
+          return '\
+            <div \
+                ng-hide="$mdAutocompleteCtrl.hidden"\
+                class="md-virtual-repeat-container md-autocomplete-suggestions-container md-whiteframe-z1' + menuContainerClass + '"\
```

Comment:
Is `md-virtual-repeat-container` needed here? Should this have `md-autocomplete-variable-height-list-container`?

I'd like to rename that to `md-autocomplete-list-container` as well.

Reply:
Yes, you're right. I got confused here. Good catch!